packages/mpg123/mpg123.0.2/opam

```diff
@@ -0,0 +1,40 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "MP3 decoding library"
+description: "OCaml bindings to libmpg123"
+maintainer: ["Michael Bacarella <m@bacarella.com>"]
+authors: ["Michael Bacarella <m@bacarella.com>"]
+license: "LGPL-2.1-or-later with OCaml-LGPL-linking-exception"
+homepage: "https://github.com/mbacarella/mpg123"
+bug-reports: "https://github.com/mbacarella/mpg123/issues"
+depends: [
+  "ocaml" {>= "4.03"}
+  "dune" {>= "2.0"}
+  "dune-configurator"
+  "conf-pkg-config"
+  "ctypes"
```

Comment:
```suggestion
  "ctypes" {> "0.13.0"}
```
This is where "Move the TYPE and FOREIGN signatures from cstubs to ctypes" happened